src/Common/tests/System/Collections/DictionaryExtensions.cs

```diff
@@ -1,20 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-namespace System.Collections.Generic
-{
-    internal static class DictionaryExtensions
```

Comment:
It's actually still needed.   Dictionary.TryAdd only exists in .NET Core, not in desktop (.NET Framework), so this extension method is used by the tests when compiled for desktop to provide a polyfill.